po/fr.po

```diff
@@ -0,0 +1,676 @@
+msgstr "Somme de contrôle SHA512"
+
+msgid "Compute SHA512 hash sum"
+msgstr "Calculer la somme de hachage SHA512"
+
+msgid "MD5 Checksum"
+msgstr "Somme de contrôle MD5"
+
+msgid "Compute MD5 hash sum"
+msgstr "Calculer la somme de hachage MD5"
+
+msgid "Base64 Encode"
+msgstr "Encodage en Base64"
+
+msgid "Encode text to Base64"
+msgstr "Encoder le texte en Base64"
+
+msgid "Base64 Decode"
+msgstr "Décodage en Base64"
+
+msgid "Decode text from Base64"
+msgstr "Décoder le texte en Base64"
+
+msgid "Remove Trailing"
+msgstr "Supprimer les fins de ligne"
+
+msgid "Remove trailing whitespaces and newlines"
+msgstr "Supprimer les espaces et sauts de ligne de fin de ligne"
+
+msgid "Trim Lines"
+msgstr "Tronquer les lignes"
+
+msgid "Remove leading and trailing whitespaces and newlines"
+msgstr "Supprimer les espaces et sauts de ligne en tête et en fin de ligne"
+
+msgid "Count Symbols"
+msgstr "Compter les symboles"
+
+msgid "Count how many symbols are in text"
+msgstr "Compter le nombre de symboles dans le texte"
+
+msgid "Count Lines"
+msgstr "Compter les lignes"
+
+msgid "Count how many lines are in text"
+msgstr "Compter le nombre de lignes dans le texte"
+
+msgid "Count Words"
+msgstr "Compter les mots"
+
+msgid "Count how many words are in text"
+msgstr "Compter le nombre de mots dans le texte"
+
+msgid "Prettify JSON"
+msgstr "Embellissement JSON"
+
+msgid "Parse JSON and prettify it"
+msgstr "Analyser le texte en JSON et l'embellir"
+
+msgid "Minify JSON"
+msgstr "Minimisation JSON"
+
+msgid "Parse JSON and minify it"
+msgstr "Analyser le texte en JSON et le réduire"
+
+msgid "Escape String"
+msgstr "Échappement de chaîne"
+
+msgid "Replace special characters with escape sequences"
+msgstr "Remplacer les caractères spéciaux par des séquences d'échappement"
+
+msgid "Escape HTML"
+msgstr "Échappement HTML"
+
+msgid "Replace special characters with HTML-safe sequences"
+msgstr "Remplacer les caractères spéciaux par des séquences sûres pour le HTML"
+
+msgid "Escape URL"
+msgstr "Échappement des URL"
+
+msgid "Replace special characters with URL-safe sequences"
+msgstr "Remplacer les caractères spéciaux par des séquences sûres pour les URL"
+
+msgid "Unescape String"
+msgstr "Déséchappement de chaîne"
+
+msgid "Replace escape sequences with characters"
+msgstr "Remplacer les séquences d'échappement par des caractères"
+
+msgid "Unescape HTML"
+msgstr "Déséchappement HTML"
+
+msgid "Replace HTML sequences with characters"
+msgstr "Remplacer les séquences HTML par des caractères"
+
+msgid "Unescape URL"
+msgstr "Déséchappement des URL"
+
+msgid "Replace URL sequences with characters"
+msgstr "Remplacer les séquences URL par des caractères"
+
+msgid "Convert JSON to YAML"
+msgstr "Conversion de JSON en YAML"
+
+msgid "Parse JSON and convert it to YAML document"
+msgstr "Analyser le texte JSON et le convertir en document YAML"
+
+msgid "Convert YAML to JSON"
+msgstr "Conversion de YAML en JSON"
+
+msgid "Parse YAML document and conver it to JSON"
```

Comment:
This typo has been fixed, it needs to be `"Parse YAML document and convert it to JSON"` now